app/classifier/tasks/dropdown/index.spec.js

```diff
@@ -1,66 +1,263 @@
+/* eslint-disable func-names, prefer-arrow-callback */
+/* global describe, it, beforeEach */
+
 import React from 'react';
 import assert from 'assert';
 import { mount } from 'enzyme';
+import Select from 'react-select'; // required to properly simulate change
 import DropdownTask from './';
+import { workflow } from '../../../pages/dev-classifier/mock-data';
 
-const task = {
+const singleSelect = {
```

Comment:
should just use multiSelect? not sure it could encompass all scenarios. I like having `singleSelect`, but not if there's too much going on.